scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SchedulerPage.java

```diff
@@ -504,6 +517,34 @@ public void execute(Boolean value) {
                                 schedResumeButton,
                                 schedPauseButton,
                                 schedKillButton);
+        // Adding tooltips on Admin actions
+        ListGridField titleFieldDefaults = adminMenu.getTitleFieldDefaults();
+        titleFieldDefaults.setShowHover(true);
+        titleFieldDefaults.setHoverCustomizer(new HoverCustomizer() {
+            @Override
+            public String hoverHTML(Object value, ListGridRecord record, int rowNum, int colNum) {
+
+                if (value.toString().equalsIgnoreCase(START))
+                    return "Start Scheduler Server from Stopped status";
+
+                else if (value.toString().equalsIgnoreCase(STOP))
+                    return "Stop Scheduler Server (Submitted Jobs terminate)";
+
+                else if (value.toString().equalsIgnoreCase(FREEZE))
+                    return "Freeze Scheduler Server (Running Tasks terminate)";
+
+                else if (value.toString().equalsIgnoreCase(RESUME))
+                    return "Resume Scheduler Server from Paused status";
+
+                else if (value.toString().equalsIgnoreCase(PAUSE))
+                    return "Pause Scheduler Server (Running Jobs terminate)";
+
+                else if (value.toString().equalsIgnoreCase(KILL))
+                    return "Kill Scheduler Server";
+
+                return null;
```

Comment:
maybe you would like to handle null case as well. but i do not insist

Reply:
nice comment... but not in that case